Libraries/Alert/AlertIOS.js

```diff
@@ -136,14 +139,15 @@ class AlertIOS {
     title: ?string,
     message?: ?string,
     callbackOrButtons?: ?(() => void) | ButtonsArray,
-    type?: AlertType,
+    tintColor?: ?string,
+    type?: AlertType
```

Comment:
This is a breaking change. Where people passed a `type` in their apps, this will be now treated as the `tintColor` and fail.